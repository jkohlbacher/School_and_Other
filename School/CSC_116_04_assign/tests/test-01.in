aab
act
abt
aeioubcd
aeiou
