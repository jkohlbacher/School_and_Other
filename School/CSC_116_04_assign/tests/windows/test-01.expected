Read 178691 words.
Tiles: [aab]
Solution: 1 [aba] Score [5]
Solution: 2 [baa] Score [5]
Tiles: [act]
Solution: 1 [act] Score [5]
Solution: 2 [cat] Score [5]
Tiles: [abt]
Solution: 1 [bat] Score [5]
Solution: 2 [tab] Score [5]
Tiles: [aeioubcd]
Solution: 1 [abduce] Score [11]
Solution: 2 [bodice] Score [11]
Solution: 3 [ceboid] Score [11]
Solution: 4 [cuboid] Score [11]
Tiles: [aeiou]
Solution: 1 [eau] Score [3]
